content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -772,3 +772,48 @@ and calls `jar` which uses
 [both `with-dir` and `with-project-root`](https://github.com/polyfy/polylith/blob/9053b190d5f3b0680ac4fe5c5f1851f7c0d40830/build.clj#L146-L147)
 to set the project root while performing `tools.deps` and `tools.build`
 operations.
+
+## Including java code in a Clojure project
+Many Clojure projects will, at some point, require the inclusion of Java files.
+Adding and using such files is simple and can be done like this:
```

Comment:
Remove "is simple and" -- tutorials should avoid telling the reader something is "simple" or "easy" (if it was, the reader probably wouldn't need the tutorial).

Reply:
Agreed. This was more of a hint towards that in the end, not many steps are required. I didn't want to imply that figuring out the steps was simple - it was not obvious to me the first time I tried.
Nonetheless, there is no advantage to the wording, so I removed it.